task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -544,3 +548,23 @@ spec:
         - cyclonedx
         - $(params.IMAGE)
       workingDir: /var/workdir
+    - name: push-dockerfile
+      image: quay.io/konflux-ci/oras@sha256:5d0a8a5535fcc4ba467264cacbdeab2fb8662a538a61cb7fc8b3155e3f20fa39
+      env:
+        - name: IMAGE_DIGEST_FILE
+          value: $(results.IMAGE_DIGEST.path)
+        - name: DOCKERFILE_PATH
+          value: /shared/Dockerfile
+      script: |
+        suffix=".dockerfile"
+        image_without_tag=${IMAGE%:*}
+        # remove the potential existing newline
+        manifest_digest=$(cat "${IMAGE_DIGEST_FILE}" | tr -d '\r\n')
+        df_image=${image_without_tag}:${manifest_digest/:/-}${suffix}
```

Comment:
Please use full word, it makes code easier to read

Reply:
Updated.